worksheet_C/readme.MD

```diff
@@ -0,0 +1,10 @@
+Worksheet C  
+A) The algorithm checks if there is a duplicate letter in the list by going through the list in order to see if a letter is repeated, if a letter has a duplicate the result will be true. 
+B) Because of the loop within the loop, so it has to check each item in the first list as well as each item in the second list so it will be N * N = N^2 
+C) Because it still checks all the numbers but it doesn't check the last number because its been check already making it almost twice as fast. Eg : 1:2 1:1 2:2 2:1 
```

Comment:
This is unclear. The idea is that the original algorithm checks each pair twice (i.e. checks both 1:2 and 2:1) whereas the new one checks only once.